std_msgs/Header header
string group_name
bool is_absolute
geometry_msgs/PoseArray poses
float64 stamp
bool allow_collision
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status